lib/logstasher/railtie.rb

```diff
@@ -16,45 +16,20 @@ class Railtie < ::Rails::Railtie
       ::LogStasher.include_parameters       = options.include_parameters
       ::LogStasher.serialize_parameters     = options.serialize_parameters
       ::LogStasher.silence_standard_logging = options.silence_standard_logging
-      ::LogStasher.logger                   = options.logger || default_logger
+      ::LogStasher.logger                   = options.logger
```

Comment:
I'm not 100% sure the order of execution of these three places:
1. The config/environments/*.rb file in the consuming project.
2. The config/initializers/logstasher.rb in the consuming project.
3. This railtie.rb file here.

Just in case 1 or 2 could happen before 3, should we change `=` to `||=` ?
```ruby
::LogStasher.logger ||= options.logger
```

(If you are confident that this file will always get executed earlier than 1 or 2, then we should be ok to leave it as is.)